gradoop-common/src/test/java/org/gradoop/common/model/impl/pojo/ElementTest.java

```diff
@@ -0,0 +1,143 @@
+package org.gradoop.common.model.impl.pojo;
+
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.junit.Test;
+import org.mockito.Mockito;
+import static junit.framework.TestCase.fail;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.CALLS_REAL_METHODS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.withSettings;
+
+public class ElementTest {
+
+  @Test
+  public void testSetId() {
+    Element elementMock = mock(Element.class, Mockito.CALLS_REAL_METHODS);
+    GradoopId id = new GradoopId();
+
+    elementMock.setId(id);
+
+    assertSame(id, elementMock.id);
+  }
+
+  @Test
+  public void testSetProperty() {
+    Element elementMock = mock(Element.class, Mockito.CALLS_REAL_METHODS);
+
+    elementMock.setProperty("key", "");
+
+    Properties properties = new Properties();
+    properties.set("key", "");
+
+    assertEquals(elementMock.properties, properties);
+  }
+
+  @Test
+  public void testSetPropertyNull() {
+    Element elementMock = mock(Element.class, Mockito.CALLS_REAL_METHODS);
+
+    try {
+      elementMock.setProperty(null);
+      fail("Expected an NullPointerException to be thrown");
+    } catch (NullPointerException nullPointerException) {
+      assertSame(nullPointerException.getMessage(), "Property was null");
```

Comment:
You can use `@Test(expected = NullPointerException.class)` here to check for exceptions.
Also I don't think you have to test the message of the exception.

Reply:
done